packages/compass-e2e-tests/tests/my-queries-tab.test.ts

```diff
@@ -40,282 +45,333 @@ async function openMenuForQueryItem(
+    collectionName,
+    'Documents'
+  );
+  await browser.runFindOperation('Documents', filter, queryOptions);
+  await browser.clickVisible(Selectors.QueryBarHistoryButton);
+
+  // Wait for the popover to show
+  const history = await browser.$(Selectors.QueryBarHistory);
+  await history.waitForDisplayed();
+
+  // wait for the recent item to show.
+  const recentCard = await browser.$(Selectors.QueryHistoryRecentItem);
+  await recentCard.waitForDisplayed();
+
+  // Save the ran query
+  await browser.hover(Selectors.QueryHistoryRecentItem);
+  await browser.clickVisible(Selectors.QueryHistoryFavoriteAnItemButton);
+  await browser.setValueVisible(
+    Selectors.QueryHistoryFavoriteItemNameField,
+    savedQueryName
+  );
+  await browser.clickVisible(Selectors.QueryHistorySaveFavoriteItemButton);
+}
+
+async function saveAggregation(
+  browser: CompassBrowser,
+  databaseName: string,
+  collectionName: string,
+  stageName: string,
+  stageText: string,
+  savedAggregationName: string
+) {
+  if (knownQueryNames[savedAggregationName]) {
+    // same comment as for saveQuery
+    throw new Error('Please use unique query names');
+  }
+  knownQueryNames[savedAggregationName] = true;
+
+  // Navigate to aggregation
+  await browser.navigateToCollectionTab(
+    DEFAULT_CONNECTION_NAME_1,
+    databaseName,
+    collectionName,
+    'Aggregations'
+  );
+  // add stage
+  await browser.clickVisible(Selectors.AddStageButton);
+  await browser.$(Selectors.stageEditor(0)).waitForDisplayed();
+  // select $match
+  await browser.focusStageOperator(0);
+  await browser.selectStageOperator(0, stageName);
+  await browser.setCodemirrorEditorValue(Selectors.stageEditor(0), stageText);
+
+  await browser.clickVisible(Selectors.SavePipelineMenuButton);
+  const menuElement = await browser.$(Selectors.SavePipelineMenuContent);
+  await menuElement.waitForDisplayed();
+  await browser.clickVisible(Selectors.SavePipelineSaveAsAction);
+
+  // wait for the modal to appear
+  const savePipelineModal = await browser.$(Selectors.SavePipelineModal);
+  await savePipelineModal.waitForDisplayed();
+
+  // set aggregation name
+  await browser.waitForAnimations(Selectors.SavePipelineNameInput);
+  await browser.setValueVisible(
+    Selectors.SavePipelineNameInput,
+    savedAggregationName
+  );
+
+  // click save button
+  const createButton = await browser
+    .$(Selectors.SavePipelineModal)
+    .$('button=Save');
+
+  await createButton.click();
+}
+
 describe('My Queries tab', function () {
   let compass: Compass;
   let browser: CompassBrowser;
 
+  let client_1: MongoClient;
+  let client_2: MongoClient;
+  /*
+  const connectionStrings = [DEFAULT_CONNECTION_STRING_1];
+  if (TEST_MULTIPLE_CONNECTIONS) {
+    connectionStrings.push(DEFAULT_CONNECTION_STRING_2);
+  }
+  clients = connectionStrings.map(
+    (connectionString) => new MongoClient(connectionString)
+  );
+
+  await Promise.all(clients.map((client) => client.connect()));
+  */
```

Comment:
leftover?

Reply:
ugh now I merged before spotting this. Will remove in the next PR that adds E2E tests.